src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+public class DukeTest {
```

Comment:
Perhaps create new classes like TaskListTest to test for methods in TaskList rather than putting everything in DukeTest